Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/bean/ConceptsMergeBean.java

```diff
@@ -0,0 +1,164 @@
+package edu.asu.conceptpower.app.bean;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import edu.asu.conceptpower.app.core.POS;
+import edu.asu.conceptpower.core.ConceptType;
+
+public class ConceptsMergeBean {
```

Comment:
I tried to use two different objects, but the JSTL spring tags don't support two different objects. One for input and another one for submitted values. For example, the <form:input> tag has only one attribute path with which the value will be bound to the server. Two objects scenario may hold good only for select boxes, where the set of objects can be iterated from a different object but the one selected value can be loaded into the submitted object. But we have POS, Concept type, concept list as a drop down, rest of the values are input or text area. Not sure how to use two different objects here (one for input and another for submitted output). In addition, I checked the ConceptEditBean, we have used the same object for both input and submitted values. Please let me know if there is a way to handle this in JSTL